laalaa/apps/advisers/middleware.py

```diff
@@ -0,0 +1,17 @@
+from django.http import HttpResponse
+
+class LivelinessMiddleware(object):
```

Comment:
It seems "readiness" might be a better name here:

> Sometimes, applications are temporarily unable to serve traffic. For example, an application might need to load large data or configuration files during startup. In such cases, you don’t want to kill the application, but you don’t want to send it requests either. Kubernetes provides readiness probes to detect and mitigate these situations. A pod with containers reporting that they are not ready does not receive traffic through Kubernetes Services.

[source](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)

"Liveness" read more like health checks to me:

> For example, liveness probes could catch a deadlock, where an application is running, but unable to make progress. Restarting a Container in such a state can help to make the application more available despite bugs

[source](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)

What do you think about renaming this class to `PingMiddleware` until we migrate to Kubernetes?

Reply:
yeap, makes sense.